app/scripts/metamask-controller.js

```diff
@@ -14,6 +14,7 @@ import { stripHexPrefix } from 'ethereumjs-util';
 import log from 'loglevel';
 import TrezorKeyring from 'eth-trezor-keyring';
 import LedgerBridgeKeyring from '@metamask/eth-ledger-bridge-keyring';
+import { MetaMaskKeyring as QRHardwareKeyring } from '@keystonehq/metamask-airgapped-keyring';
```

Comment:
Is `'@keystonehq/metamask-airgapped-keyring'` accessible on github? Or is it private?

Reply:
You can find it here: https://github.com/KeystoneHQ/keystone-airgaped-base/tree/master/packages/metamask-airgapped-keyring